src/main/java/Event.java

```diff
@@ -0,0 +1,10 @@
+public class Event extends Deadline{
+    private final String start;
```

Comment:
Suggestion: Leave space between initialising variables and class methods for neater presentation